internal/website/content/howto/pubsub/subscribe.md

```diff
@@ -7,12 +7,14 @@ toc: true
 
 Subscribing to messages on a topic with the Go CDK takes three steps:
 
-1. Open the subscription with the Pub/Sub provider of your choice (once per
+1. [Open the subscription][] with the Pub/Sub provider of your choice (once per
```

Comment:
I think "Open a subscription" is more grammatically correct

Reply:
Just took it from older version :)

 Sure, Done.